### ISSUE TITLE
Homey disconnects on opening heavy insight via usb ethernet/RNDIS Gadget

### ISSUE THREAD

**Issue by nattlip** [athombv/homey#2345]
Homey Firmware
1.5.6 1.5.7 1.5.8

**What did you try to achieve?**
opning a heavy loaded insight while using the usb port , the one i use to debug over usb

*what did you expect*
see the insight on my screen 

nothing happens after loading the correct template, and homey becomes offline for at last an hour  in my.athom.com. the get contact i have to ptp . loading the insight via the wifi ethernet works very slow but it works. every other homey screen works perfect over usb.

**Comment by Dijker** [comment]
Does it work if you don't use the USB developers interface?

**Comment by nattlip** [comment]
you mean over wifi , yes i stated that , i dont always debug apps but homey is always connected via usb, so indeed it doent matter if you are debugging or not

**Comment by Dijker** [comment]
So actually it doesn't matter if u use USB or not and confirm the issue #2110 
Also happens on the USB developers interface?

**Comment by nattlip** [comment]
yes the developr interface also disconnects when used. issue #2110 is a temporary disconnect , mine is perpetual , tested for morte then 1 hour, which is to long for a device controlling via tcp
